src/Microsoft.Azure.WebJobs.Host/Bindings/FunctionBinding.cs

```diff
@@ -25,7 +25,9 @@ public FunctionBinding(FunctionDescriptor descriptor, IReadOnlyDictionary<string
         {
             Dictionary<string, IValueProvider> results = new Dictionary<string, IValueProvider>();
 
-            BindingContext bindingContext = new BindingContext(context, null);
+            // Supplied bindings can be direct parameters or route parameters. 
```

Comment:
FunctionBinding is used for functions w/o trigger bindings only. Doesn't that mean that what you're trying to do here won't work for triggered functions?